yarn-project/circuits.js/src/contract/contract_class_id.ts

```diff
@@ -56,8 +57,19 @@ export type ContractClassIdPreimage = {
   publicBytecodeCommitment: Fr;
 };
 
-// TODO(#5860): Replace with actual implementation
```

Comment:
This is removed here, but there are a few more instances which need to be fixed before this issue can be considered "done"